diff --git a/node_modules/docx/dist/index.cjs b/node_modules/docx/dist/index.cjs
index 87c6a82..d581053 100644
--- a/node_modules/docx/dist/index.cjs
+++ b/node_modules/docx/dist/index.cjs
@@ -29417,14 +29417,7 @@ var require_jszip_min = /* @__PURE__ */ __commonJSMin(((exports, module) => {
 						s.observe(a, { characterData: !0 }), r = function() {
 							a.data = i = ++i % 2;
 						};
-					} else if (t.setImmediate || void 0 === t.MessageChannel) r = "document" in t && "onreadystatechange" in t.document.createElement("script") ? function() {
-						var e = t.document.createElement("script");
-						e.onreadystatechange = function() {
-							u(), e.onreadystatechange = null, e.parentNode.removeChild(e), e = null;
-						}, t.document.documentElement.appendChild(e);
-					} : function() {
-						setTimeout(u, 0);
-					};
+					} else if (t.setImmediate || void 0 === t.MessageChannel) r = function() { setTimeout(u, 0); };
 					else {
 						var o = new t.MessageChannel();
 						o.port1.onmessage = u, r = function() {
@@ -31012,14 +31005,7 @@ while (n === a[++i] && n === a[++i] && n === a[++i] && n === a[++i] && n === a[+
 								c(e.data);
 							}, function(e) {
 								t.port2.postMessage(e);
-							}) : l && "onreadystatechange" in l.createElement("script") ? (s = l.documentElement, function(e) {
-								var t = l.createElement("script");
-								t.onreadystatechange = function() {
-									c(e), t.onreadystatechange = null, s.removeChild(t), t = null;
-								}, s.appendChild(t);
-							}) : function(e) {
-								setTimeout(c, 0, e);
-							}, e.setImmediate = function(e) {
+							}) : function(e) { setTimeout(c, 0, e); }, e.setImmediate = function(e) {
 								"function" != typeof e && (e = new Function("" + e));
 								for (var t = new Array(arguments.length - 1), r = 0; r < t.length; r++) t[r] = arguments[r + 1];
 								return h[o] = {
diff --git a/node_modules/docx/dist/index.iife.js b/node_modules/docx/dist/index.iife.js
index a8646c5..d3ca195 100644
--- a/node_modules/docx/dist/index.iife.js
+++ b/node_modules/docx/dist/index.iife.js
@@ -29418,14 +29418,7 @@ MAX: 9026 };
 							s.observe(a, { characterData: !0 }), r = function() {
 								a.data = i = ++i % 2;
 							};
-						} else if (t.setImmediate || void 0 === t.MessageChannel) r = "document" in t && "onreadystatechange" in t.document.createElement("script") ? function() {
-							var e = t.document.createElement("script");
-							e.onreadystatechange = function() {
-								u(), e.onreadystatechange = null, e.parentNode.removeChild(e), e = null;
-							}, t.document.documentElement.appendChild(e);
-						} : function() {
-							setTimeout(u, 0);
-						};
+						} else if (t.setImmediate || void 0 === t.MessageChannel) r = function() { setTimeout(u, 0); };
 						else {
 							var o = new t.MessageChannel();
 							o.port1.onmessage = u, r = function() {
@@ -31013,14 +31006,7 @@ while (n === a[++i] && n === a[++i] && n === a[++i] && n === a[++i] && n === a[+
 									c(e.data);
 								}, function(e) {
 									t.port2.postMessage(e);
-								}) : l && "onreadystatechange" in l.createElement("script") ? (s = l.documentElement, function(e) {
-									var t = l.createElement("script");
-									t.onreadystatechange = function() {
-										c(e), t.onreadystatechange = null, s.removeChild(t), t = null;
-									}, s.appendChild(t);
-								}) : function(e) {
-									setTimeout(c, 0, e);
-								}, e.setImmediate = function(e) {
+								}) : function(e) { setTimeout(c, 0, e); }, e.setImmediate = function(e) {
 									"function" != typeof e && (e = new Function("" + e));
 									for (var t = new Array(arguments.length - 1), r = 0; r < t.length; r++) t[r] = arguments[r + 1];
 									return h[o] = {
diff --git a/node_modules/docx/dist/index.mjs b/node_modules/docx/dist/index.mjs
index c023db4..83ce145 100644
--- a/node_modules/docx/dist/index.mjs
+++ b/node_modules/docx/dist/index.mjs
@@ -29420,14 +29420,7 @@ var require_jszip_min = /* @__PURE__ */ __commonJSMin(((exports, module) => {
 						s.observe(a, { characterData: !0 }), r = function() {
 							a.data = i = ++i % 2;
 						};
-					} else if (t.setImmediate || void 0 === t.MessageChannel) r = "document" in t && "onreadystatechange" in t.document.createElement("script") ? function() {
-						var e = t.document.createElement("script");
-						e.onreadystatechange = function() {
-							u(), e.onreadystatechange = null, e.parentNode.removeChild(e), e = null;
-						}, t.document.documentElement.appendChild(e);
-					} : function() {
-						setTimeout(u, 0);
-					};
+					} else if (t.setImmediate || void 0 === t.MessageChannel) r = function() { setTimeout(u, 0); };
 					else {
 						var o = new t.MessageChannel();
 						o.port1.onmessage = u, r = function() {
@@ -31015,14 +31008,7 @@ while (n === a[++i] && n === a[++i] && n === a[++i] && n === a[++i] && n === a[+
 								c(e.data);
 							}, function(e) {
 								t.port2.postMessage(e);
-							}) : l && "onreadystatechange" in l.createElement("script") ? (s = l.documentElement, function(e) {
-								var t = l.createElement("script");
-								t.onreadystatechange = function() {
-									c(e), t.onreadystatechange = null, s.removeChild(t), t = null;
-								}, s.appendChild(t);
-							}) : function(e) {
-								setTimeout(c, 0, e);
-							}, e.setImmediate = function(e) {
+							}) : function(e) { setTimeout(c, 0, e); }, e.setImmediate = function(e) {
 								"function" != typeof e && (e = new Function("" + e));
 								for (var t = new Array(arguments.length - 1), r = 0; r < t.length; r++) t[r] = arguments[r + 1];
 								return h[o] = {
diff --git a/node_modules/docx/dist/index.umd.cjs b/node_modules/docx/dist/index.umd.cjs
index 64b854e..0c387f1 100644
--- a/node_modules/docx/dist/index.umd.cjs
+++ b/node_modules/docx/dist/index.umd.cjs
@@ -29420,14 +29420,7 @@ MAX: 9026 };
 							s.observe(a, { characterData: !0 }), r = function() {
 								a.data = i = ++i % 2;
 							};
-						} else if (t.setImmediate || void 0 === t.MessageChannel) r = "document" in t && "onreadystatechange" in t.document.createElement("script") ? function() {
-							var e = t.document.createElement("script");
-							e.onreadystatechange = function() {
-								u(), e.onreadystatechange = null, e.parentNode.removeChild(e), e = null;
-							}, t.document.documentElement.appendChild(e);
-						} : function() {
-							setTimeout(u, 0);
-						};
+						} else if (t.setImmediate || void 0 === t.MessageChannel) r = function() { setTimeout(u, 0); };
 						else {
 							var o = new t.MessageChannel();
 							o.port1.onmessage = u, r = function() {
@@ -31015,14 +31008,7 @@ while (n === a[++i] && n === a[++i] && n === a[++i] && n === a[++i] && n === a[+
 									c(e.data);
 								}, function(e) {
 									t.port2.postMessage(e);
-								}) : l && "onreadystatechange" in l.createElement("script") ? (s = l.documentElement, function(e) {
-									var t = l.createElement("script");
-									t.onreadystatechange = function() {
-										c(e), t.onreadystatechange = null, s.removeChild(t), t = null;
-									}, s.appendChild(t);
-								}) : function(e) {
-									setTimeout(c, 0, e);
-								}, e.setImmediate = function(e) {
+								}) : function(e) { setTimeout(c, 0, e); }, e.setImmediate = function(e) {
 									"function" != typeof e && (e = new Function("" + e));
 									for (var t = new Array(arguments.length - 1), r = 0; r < t.length; r++) t[r] = arguments[r + 1];
 									return h[o] = {
